plugins/Referrers/lang/en.json

```
@@ -69,6 +70,8 @@
         "SocialsReportDocumentation": "This report shows which social networks led visitors to your website.<br />By clicking on a row in the table, you can see from which social network pages visitors came to your website.",
         "SocialsSubcategoryHelp": "In this table, you can see which websites referred visitors to your site.",
         "AIAssistants": "AI Assistants",
+        "AIAssistantsReportDocumentation": "This report shows which AI Assistants led visitors to your website.<br />By clicking on a row in the table, you can see from which AI Assistant pages visitors came to your website.",
```

```suggestion
        "AIAssistantsReportDocumentation": "This report shows which AI Assistants led visitors to your website. %s By clicking on a row in the table, you can see from which AI Assistant pages visitors came to your website.",
```

Ignoring (A|a) case, the `br />` should be a placeholder to match the other entries and the usage that passes it to the translator.